data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2593,6 +2599,35 @@ spec:
                     description: ExternalNetwork is name of the external network in
                       your OpenStack cluster.
                     type: string
+                  failureDomains:
+                    description: FailureDomains contains a list of domains and their
+                      specifications.
+                    items:
+                      description: FailureDomain stores the configuration for
+                        the domain.
+                      properties:
+                        name:
+                          description: Name is the name of the failure domain.
+                          type: string
+                        computeZone:
+                          description: Name of the availability zone where the instances
+                            should be deployed. If no zone is provided, all instances
+                            will be deployed on OpenStack Nova default availability zone.
+                          type: string
+                        storageZone:
+                          description: Name of the availability zone where the volumes
+                            should be deployed. If no zone is provided, all instances
+                            will be deployed on OpenStack Cinder default availability zone.
+                          type: string
+                        subnet:
+                          description: OpenStack Neutron subnet UUID where the instances
+                            should be connected to as a primary interface.
+                          type: string
+                      required:
+                      - name
+                      - subnet
```

Comment:
I think we should not require any field inside FailureDomain, so that they can be used as a drop-in replacement for `zones` for example.